flink-python/pyflink/fn_execution/beam/beam_operations_slow.py

```diff
@@ -162,23 +165,36 @@ def __init__(self, name, spec, counter_factory, sampler, consumers):
         super(DataStreamStatelessFunctionOperation, self).__init__(name, spec, counter_factory,
                                                                    sampler, consumers)
 
-    def generate_func(self, udfs):
-        func = operation_utils.extract_data_stream_stateless_funcs(udfs=udfs)
-        return lambda it: map(func, it), []
+    def open_func(self):
+        for user_defined_func in self.user_defined_funcs:
+            runtime_context = RuntimeContext(
```

Comment:
Can we reuse the runtime_context by creating it outside of the for loop?

Reply:
Actually there is only one user-defined function for Python DataStream API. Besides, I think that the runtime context may not be shared even if there are multiple user-defined functions, e.g. when we support metrics, each function may have different metrics group. So what about keeping it as it's now and reconsider whether we can share them when there are actually multiple user-defined functions.